scripts/pallindrome.py

```diff
@@ -0,0 +1,15 @@
+def is_symmetrical(input_str):
+    input_str = input_str.lower()  # Convert to lowercase for case-insensitive comparison
+    length = len(input_str)
+    midpoint = length // 2
+
+    for i in range(midpoint):
+        if input_str[i] != input_str[length - i - 1]:
+            return False
+    return True
+
+user_input = input("Enter a string: ")
```

Comment:
I really love what you did! Could you please add lines 11 to 15 into a main() function call it when the script runs:

```
if __name__ == '__main__':
    main()
```

Just to be consistent with the rest of the repo please? 🙂

Reply:
Sure!